src/Microsoft.DotNet.Interactive.CSharpProject/ProjectFile.cs

```diff
@@ -1,16 +1,23 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.IO;
+
 namespace Microsoft.DotNet.Interactive.CSharpProject;
 
 public class ProjectFile
 {
-    public string Path { get; }
+    public string RelativePath { get; }
```

Comment:
Why not the `dotnet try` `RelativeFilePath` type?

Reply:
is used via the constructor but exposed as string